src/pages/home.tsx

```
@@ -14,18 +14,57 @@ import { HomeChatInput } from "@/components/chat/HomeChatInput";
   const { streamMessage } = useStreamChat({ hasChatId: false });
   const posthog = usePostHog();
+  const appVersion = useAppVersion();
+  const [releaseNotesOpen, setReleaseNotesOpen] = useState(false);
+  const [releaseUrl, setReleaseUrl] = useState("");
+  const { theme } = useTheme();
+
+  useEffect(() => {
+    const updateLastVersionLaunched = async () => {
+      if (
+        appVersion &&
+        appVersion.match(/^\d+\.\d+\.\d+$/) &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)